lib/elixir/lib/kernel.ex

```diff
@@ -4284,7 +4285,7 @@ defmodule Kernel do
   belongs to the struct:
 
       %User{name: "john", unknown: :key}
-      ** (KeyError) key :unknown not found in: %User{age: 21, name: nil}
+      #=> ** (KeyError) key :unknown not found in: %User{age: 21, name: nil}
```

Comment:
Same here and the exceptions below.

Reply:
we were using this method for code that couldn't be put in doctests because otherwise the module wouldn't compile

Reply:
somehow we need to differentiate what is input and what is output.
in doctests input is represented by `iex>`,

Reply:
I am aware. The only reason we use `#=>` is because it may not be clear to folks that `true` is a return value in something like:

    User.is_foo?(foo)
    true

So we do:

    User.is_foo?(foo)
    #=> true

But an exception is always clear that it is an exception. Adding #=> actually makes it confusing, because it is no longer clear if it is a return value or the exception. Using the same format as in doctest is actually a positive thing.

Reply:
I agree, but I'd prefer to have the same style throughout the docs. the are other parts were we are using this, should we remove the `#=>` when an exception is thrown?